code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -786,13 +786,20 @@
 	metabolization_rate = 0.25 * REAGENTS_METABOLISM
 	overdose_threshold = 30
 
+/datum/reagent/medicine/epinephrine/on_mob_add(mob/living/carbon/M)
+	..()
+	M.add_trait(TRAIT_NOCRITDAMAGE, id)
+
```

Comment:
```suggestion
if(M.health < 0)
		M.adjustOxyLoss(-20, 0)
```

Reply:
good catch, `M.health < M.critThreshold` should be used here.

Reply:
for the record i dont know if that is the real var name

Reply:
there's a var like that idk if it's on the mob tho, used to be on species but got taken out in a later PR (iirc)

Reply:
`crit_threshold` is the var.

Reply:
also it should be `M.health <= M.crit_threshold`